chunk/download.go

```diff
@@ -97,7 +102,11 @@ func downloadFromAPI(client *http.Client, request *Request, buffer []byte, delay
 		Log.Debugf("%v", err)
 		return fmt.Errorf("Could not create request object %v (%v) from API", request.object.ObjectID, request.object.Name)
 	}
-
+	if request.acknowledgeAbuse {
+		q := req.URL.Query()
+		q.Add("acknowledgeAbuse", "true")
+		req.URL.RawQuery = q.Encode()
+	}
```

Comment:
Instead of decoding the query of the request, adding a query param and then reencoding it, why not store a copy of `DownloadURL` and append `"&acknowledgeAbuse=true"` to it, before building the http request?

We know that all download urls already have at least "alt=media" query param, so no need to handle different cases.

Reply:
It felt less hacky than appending a fixed string to the URL, but it's not a strong opinion I have. 
 12a93020718bddec7427647e9d69636b101a2e08 changes this to concatenate the fixed string query parameter to the URL